charts/topolvm/README.md

```diff
@@ -83,7 +83,8 @@ You need to configure kube-scheduler to use topolvm-scheduler extender by referr
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | cert-manager.enabled | bool | `false` | Install cert-manager together. # ref: https://cert-manager.io/docs/installation/kubernetes/#installing-with-helm |
-| controller.additionalLabels | object | `{}` | Additional labels to be set on the controller pod |
+| controller.podLabels | object | `{}` | Additional labels to be set on the controller pod |
+| controller.labels | object | `{}` | Additional labels to be added to the Deployment |
```

Comment:
This file is automatically generated by the `helm-docs` command. Please follow this instruction.

https://github.com/topolvm/topolvm/blob/07779ec247087d395288515dd16f2065edf98d19/charts/topolvm/README.md#update-readme

Reply:
My apologies, I missed that. 👍